collections/ansible_collections/cloudkit/service/roles/publish_templates/tasks/main.yaml

```diff
@@ -0,0 +1,43 @@
+- name: Check for existing templates
+  loop: "{{ cloudkit_templates }}"
+  loop_control:
+    label: "{{ item.id }}"
+  ansible.builtin.uri:
+    url: "{{ publish_templates_api_endpoint }}/{{ item.id }}"
```

Comment:
don't we have an endpoint to list the templates instead of getting them one by one? It would also help for the deletion/deprecation. Anyway, I don't think it's a blocker for the moment.

Reply:
> don't we have an endpoint to list the templates instead of getting them one by one?

We do, and I've made that change to the playbook. Take a look and let me know if that seems reasonable.